packages/pricing/src/services/pricing-module.ts

```diff
@@ -247,6 +249,236 @@ export default class PricingModuleService<
     return Array.isArray(data) ? results : results[0]
   }
 
+  async addRules(
```

Comment:
A general question - why is this called `addRules` and not `createRules` so we are consistent?

Reply:
I must say that i have no idea, there is plenty of improvements left to do. I haven't focused on naming here but indeed consistency is missing here, probably because it was done before the auto generated methods